### ISSUE TITLE
Feet no longer on ground after applying legs' length related measure targets

### ISSUE THREAD

**Issue by maryam-fatima** [makehumancommunity/mpfb2#172]
**Describe the bug**
The human model doesn't translate to make its feet touch the ground after the value for lower leg_length, upper leg_length measure targets  is updated. **Is this intentional?** The other length and height related measure don't cause this issue as changes in those are towards +z axis, if that makes sense.

It would be great If we have an option for "feet_on_ground" in UI.

**Screenshots / logs**
<img width="411" alt="image" src="https://github.com/makehumancommunity/mpfb2/assets/62232940/a34f9382-4363-42f1-833a-4190ca0e4227">

<img width="411" alt="image" src="https://github.com/makehumancommunity/mpfb2/assets/62232940/03363b19-05b9-4a77-be1b-6d5b885e8f47">

**Platform and versions**
 - OS: Mac
 - Version of Blender: 4.0.2
 - Version of MPFB: blender4_measure

**Comment by joepal1976** [comment]
No, this is for least-surprise reasons and because most targets are sided. 

The least surprise part is that you might have positioned the object for some reason, and if it shifts around automatically this might mess things up. However, if you save a character with a measure target applied and then open it "from presets", I think that the create_human routine will position the feet on ground. 

The second part is that for example the non-measure leg length is sided. If the right leg is longer than the left, which foot should be on the ground? My thinking was that it'd be better to leave this to the user. 

This said: I like your PR, and see no particular reason for not merging it, barring the comment I added in the PR. 

An evolution of this would be having a feet-on-ground check box in the model panel for automatically translating the body whenever a slider is changed. This should not be a particularly heavy operation.

**Comment by wweennel** [comment]
The same request. When a reference is set, not being able to automatically keep "feet-on-ground" causes a lot of trouble. Because of the constant manual alignment.
